ObjCVideoQuickstart/ViewController.m

```diff
@@ -24,11 +24,11 @@ @interface ViewController () <UITextFieldDelegate, TVIParticipantDelegate, TVIRo
 @property (nonatomic, strong) TVILocalVideoTrack *localVideoTrack;
 @property (nonatomic, strong) TVILocalAudioTrack *localAudioTrack;
 @property (nonatomic, strong) TVIParticipant *participant;
+@property (nonatomic, strong) TVIVideoView *remoteView;
```

Comment:
Food for thought, do we want to have a strong reference to `remoteView` when UIKit will do that for us?